src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps  # pylint: disable=unused-import
+
+# pylint: disable=line-too-long
+helps['consumption'] = """
+    type: group
+    short-summary: Manage Azure Consumption.
+"""
+
+helps['consumption usage list'] = """
```

Comment:
this is the first API release for this RP. We are adding more APIs and more commands in future.